Tasks/AzureFileCopy/task.json

```diff
@@ -1,4 +1,4 @@
-﻿{
```

Comment:
Oh ok cool. Just wanted to make sure the customer doesn't run into the confusing issue when simply trying to run their task via the handler.